projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -283,17 +283,13 @@ const ProductCard = props => {
 					/>
 				) }
 			</div>
-			{
-				// If is not active, no reason to use children
-				// Since we want user to take action if isn't active
```

Comment:
> // If is not active, no reason to use children
> // Since we want user to take action if isn't active

We want to render the Children component even when the product isn't active to implement a custom UI, es described in the issue.
Please let us know if you consider it could introduce other issues.

cc @renatoagds